tools/breaking-change-detector/docs/generator.go

```diff
@@ -26,7 +27,7 @@ func Generate(outputPath string) {
 	if outputPath == "" {
 		fmt.Printf("%v", contents.String())
 	} else {
-		outName := "breaking-changes.md"
+		outName := constants.BreakingChangeFileName
```

Comment:
Was this added in a separate PR?

Reply:
I manually committed the file previously, correct.